code/modules/mob/living/taste.dm

```diff
@@ -25,7 +25,7 @@
 			text_output = pick("spiders","dreams","nightmares","the future","the past","victory",\
 			"defeat","pain","bliss","revenge","poison","time","space","death","life","truth","lies","justice","memory",\
 			"regrets","your soul","suffering","music","noise","blood","hunger","the american way")
-		if(text_output != last_taste_text || last_taste_time + 100 < world.time)
+		if((text_output != last_taste_text || last_taste_time + 100 < world.time) && taste_sensitivity < 101) // 101 means literally no taste ability.
```

Comment:
Change that 101 to a define. I'd make that -1. `#define TASTE_SENSITIVITY_NO_TASTE -1`
Then add that to all species and make a `==` check here.